Tests/SparkConnectTests/SparkSessionTests.swift

```diff
@@ -75,6 +75,16 @@ struct SparkSessionTests {
     await spark.stop()
   }
 
+  @Test
+  func table() async throws {
+    let tableName = UUID().uuidString.replacingOccurrences(of: "-", with: "")
+    let spark = try await SparkSession.builder.getOrCreate()
+    #expect(try await spark.sql("CREATE TABLE \(tableName) AS VALUES (1), (2)").count() == 0)
```

Comment:
Is it possible to test view currently?

Reply:
Thank you for review. For views, there are multiple variants like Permanent View, Temporary View, Global Temporary view. So, I postpone the test coverage until now. However, it will be added soon because I'm implementing `Catalog`, @viirya .